imgcreate/creator.py

```diff
@@ -36,14 +37,15 @@
 import dnf
 import rpm
 
-from imgcreate.errors import *
-from imgcreate.fs import *
-from imgcreate.dnfinst import *
-from imgcreate import kickstart
+from errors import *
+from fs import *
+from dnfinst import *
+from kickstart import *
```

Comment:
Please revert these changes, as they are intended to make it obvious that these modules are internal to livecd-tools.